packages/core/nuxt-module/src/index.ts

```diff
@@ -0,0 +1,130 @@
+// TODO proper bundling, for now it's just to experiment with nuxt modules api
+import fs from 'fs';
+import path from 'path';
+import log from './helpers/log';
+import merge from './helpers/merge';
+import resolveDependency from './helpers/resolveDependency';
+import performanceModule from './modules/performance';
+import storefrontUiModule from './modules/storefront-ui';
+import rawSourcesModule from './modules/raw-sources-loader';
+
+export { default as meta } from '../package.json';
+
+export interface ModuleOptions {
```

Comment:
Can we move types to the dedicated file?

Reply:
Moved